backend/middleware/authenticateToken.js

```diff
@@ -1,33 +1,29 @@
-const jwt = require("jsonwebtoken")
```

Comment:
It looks like this file doesn't seem to have any noticeable difference from the old one if I'm not mistaken. For files that we don't modify, we shouldn't be pushing them either.